crates/rolldown/tests/esbuild/default/string_export_names_iife/artifacts.snap

```diff
@@ -56,6 +57,7 @@ source: crates/rolldown_testing/src/integration_test.rs
 const all the stuff = __foo;
 const { some import: someImport } = __foo;
 
+
```

Comment:
The output actually has syntax error, the bundle result is incorrect, and the extra line because the  `__require` runtime is introduced in dist/out.js, and removed in `snapshot`, but newline is not trimmed.